### DISCUSSION TITLE
AppRailTile width

### DISCUSSION THREAD

**Discussion by jeromecc** [skeletonlabs/skeleton#1200] [Q&A]
Hi!
I'm using the exact same code of skeleton.dev for my AppRail but the labels of my tiles contain more characters. This results in the end of the labels being hidden under the 2nd column (menuNavLinks).

How could I increase the width of the AppRail tiles and/or make the text of the label wrap correctly? I tried using the width prop but the results are perplexing. For instance, adding width=w-14 to the AppRail element makes the tiles' width smaller and w-15 make them way too big (the text of the label becomes unwrapped).

My SideBar (i renamed it from DocsSidebar to Sidebar) is here: https://github.com/medica-im/healthcenter/blob/v2/frontend/src/components/Sidebar/Sidebar.svelte

Thanks a lot! :))

**Comment by endigo9740** [comment]
@jeromecc, our App Rail is based on the Material Design "Navigation Rail". They provide some great guidance in this regard:
https://m3.material.io/components/navigation-rail/guidelines#b59dc6d4-54fd-4e94-8243-838505acbaf4

Long story short, they recommend reducing the text size to fit labels that are too wide.
